product/builder-basics.mdx

```diff
@@ -10,6 +10,23 @@ The Visual Builder is composed of 4 sections:
 
 <VisualBuilderOverview />
 
+## Build & Interact Modes
+
+In the Component Toolbar you'll notice two options that dictate how you will engage with the elements on your Canvas; **Build** mode and **Interact** mode
+
+### Build
```

Comment:
I'm debating about calling this **Build Mode** instead of **Build** mode. Any thoughts? It feels like those should go together as the proper noun?